features/step_definitions/apicast_steps.rb

```diff
@@ -6,3 +6,8 @@
     .to_return(status: 200, body: "{\"policies\":{\"apicast\":[{\"description\":\"Main functionality of APIcast.\",\"$schema\":\"http:\\/\\/apicast.io\\/policy-v1\\/schema#manifest#\",\"name\":\"APIcast policy\",\"configuration\":{\"properties\":{},\"type\":\"object\"},\"version\":\"builtin\"}]}}",
                headers: { 'Content-Type' => 'application/json' })
 end
+
+Given(/^apicast registry is invalid$/) do
```

Comment:
This title imo makes me think that it can be invalid in many ways but the content implies that it is just empty 🤔 
Not a big deal though

Reply:
It's either this or creating 2 features, one for url null and other for url invalid. I think the outcome is the same, is it?

Reply:
Not, it is good with just one, when we need another one, maybe we can just pass a parameter to this one and that's it 😄

Reply:
In the end I changed it for "undefined" because that's what the issue is about, really.